packages/vector_graphics_compiler/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   path_parsing: ^1.0.1
   xml: ^6.3.0
   vector_graphics_codec: 1.1.8
+  path: ^1.8.3
```

Comment:
Doesn't this add a dependency on path to all users of flutter_svg?

Reply:
ahhh I think we cheated on this before and put it in dev_dependencies...

Reply:
We could also use a more relaxed constraint?

Reply:
I think you can cheat if its only used in the bin directory, might want to double check. I'd be fine with that.

Reply:
If that doesn't work, I'd be fine with a loose constraint like we have

Reply:
I changed this to ^1.8.0. I'll fix up the analysis issues unrelated to this patch separately.